src/core/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -183,7 +205,6 @@
             <StackPanel Orientation="Horizontal">
                 <AppBarButton Icon="Add"
                               x:Name="AddSizeButton"
-                              Width="560"
```

Comment:
I'd go for MinWidth instead of removing it entirely to make sure it renders properly when setting display scaling to e.g. 100%/150%